charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses/nutanix-cluster-class.yaml

```diff
@@ -155,7 +155,7 @@ spec:
                       valueFrom:
                         fieldRef:
                           fieldPath: spec.nodeName
-                    - name: vip_cidr
+                    - name: vip_subnet
```

Comment:
This is a breaking change in kube-vip but will handled for users using a new versioned ClusterClass and rebasing clusters on top of that.